graphix/pattern.py

```diff
@@ -1421,6 +1417,16 @@ def to_qasm3(self, filename):
                 for line in cmd_to_qasm3(command):
                     file.write(line)
 
+    def copy(self):
```

Comment:
Please do not omit return type annotation.

Reply:
Fixed in b572ce5.